packages/schema/parse/src/extract/index.ts

```diff
@@ -9,7 +9,7 @@ export type SchemaExtractor = (astNode: DocumentNode, typeInfo: TypeInfo) => voi
 
 export const extractors: SchemaExtractor[] = [
   extractUserTypes,
-  extractQueryTypes,
   extractImportedObjectTypes,
+  extractQueryTypes,
```

Comment:
Is the order of extractors important for something else?
I need extractor to parse importedObjectTypes so I can recognize it is a object in `NamedType` parsing.

Reply:
Ordering should be independent, as the extractors **shouldn't** require TypeInfo data from other extractors. This is of course an ideal scenario, and may break down in practice. Let me know if you there are dependencies between the extractors and we can figure out how to best handle this.

Reply:
Have dependecy here https://github.com/Web3-API/prototype/blob/fa6e10dad19e8b53b7c90117b3e87c88551aa4a7/packages/schema/parse/src/extract/query-utils.ts#L36

To recognize if it is object or scalar in named type extraction I need to check if the objects is defined somewhere
as I don't have `kind` from node.
Only way I found out was to check if that object exists in already extracted objects in `typeInfo`.

Reply:
Chatted about this, will be handled in another PR.